documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -0,0 +1,210 @@
+# Install the Clear Containers runtime on RHEL 7.3
+
+
+## Introduction
+
+Installing Clear Containers on RHEL is slightly more difficult than on other distributions mainly due to the fact that it does not provide current versions of gcc and glibc.
```

Comment:
Actually, "glibc" isn't a binary so should not have backticks since we're then referring to the project name (we could even refer to "the GNU C Library" instead?)

@rcaballeromx - What's your view on this? Should we specify project names all time time, or only when it's not obvious from the name of the binary which project it's from (in which case we'd mention both the binary name and the project)?